### ISSUE TITLE
Low contrast in setting page (light mode)

### ISSUE THREAD

**Issue by Huy-Ngo** [libredirect/libredirect#27]
I find the current heading color (#fbc117) to be quite low contrast (1.33:1), which is quite hard to read.
[WCAG Standard](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast) states that the contrast ratio should be at least 3:1.

I would suggest some color with better contrast on background like #0b5394, though I haven't found a good looking yellowish color that is easy to read in light mode.

**Comment by ManeraKai** [comment]
<img width=230 src="https://user-images.githubusercontent.com/40805353/153034097-abc08062-25d6-4630-976b-fda9d3a8fa74.png" />

![image](https://user-images.githubusercontent.com/40805353/153033883-cf030d59-5bac-4866-b67b-9e0e434ee4de.png)
So..?

**Comment by Huy-Ngo** [comment]
This one looks nice and more readable :+1:
